### DISCUSSION TITLE
Update nuget now

### DISCUSSION THREAD

**Discussion by alexey-tyulkin** [docopt/docopt.net#135] [General]
Hi
Can you update nuget now or soon? I want the new assembly without #32 issue.
Thanks

**Comment by atifaziz** [comment]
I can try and aim to publish a new version this month.

**Comment by atifaziz** [comment]
@alexey-tyulkin A [new version (0.7.0)](https://github.com/docopt/docopt.net/releases/tag/v0.7.0) has been published that contains the fix for bug #32.
